validators/validations.js

```diff
@@ -89,5 +89,53 @@ function validateForm(formBody) {
 
   return formPageSchema.validate(formBody)
 }
+function validateUpdateForm(formBody) {
+  // Define a Joi schema for the form input options
+  const inputOptionSchema = Joi.object({
+    label: Joi.string().required(),
+    value: Joi.string().required(),
```

Comment:
is validateForm not enough for this? I think due to the similar object data. We can look into using same function for this.